packages/sdk/fastly/__tests__/api/EdgeFeatureStore.test.ts

```diff
@@ -0,0 +1,125 @@
+import { AsyncStoreFacade, LDFeatureStore } from '@launchdarkly/js-server-sdk-common';
+
+import { EdgeFeatureStore } from '../../src/api/EdgeFeatureStore';
+import mockEdgeProvider from '../../src/utils/mockEdgeProvider';
+import * as testData from './testData.json';
+
+describe('EdgeFeatureStore', () => {
+  const sdkKey = 'sdkKey';
+  const kvKey = `LD-Env-${sdkKey}`;
+  const mockLogger = {
+    error: jest.fn(),
+    warn: jest.fn(),
+    info: jest.fn(),
+    debug: jest.fn(),
+  };
+  const mockGet = mockEdgeProvider.get as jest.Mock;
+  let featureStore: LDFeatureStore;
+  let asyncFeatureStore: AsyncStoreFacade;
+
+  beforeEach(() => {
+    mockGet.mockImplementation(() => Promise.resolve(JSON.stringify(testData)));
+    featureStore = new EdgeFeatureStore(mockEdgeProvider, sdkKey, 'MockEdgeProvider', mockLogger);
+    asyncFeatureStore = new AsyncStoreFacade(featureStore);
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('get', () => {
+    test('get flag', async () => {
+      const flag = await asyncFeatureStore.get({ namespace: 'features' }, 'testFlag1');
+
+      expect(mockGet).toHaveBeenCalledWith(kvKey);
+      expect(flag).toMatchObject(testData.flags.testFlag1);
+    });
+
+    test('invalid flag key', async () => {
```

Comment:
Then this becomes 
```suggestion
    it('returns undefined for an invalid flag key', async () => {
```

And so on for these tests.